solr/core/src/java/org/apache/solr/cluster/placement/impl/AttributeFetcherImpl.java

```diff
@@ -33,18 +37,18 @@
 import java.lang.invoke.MethodHandles;
 import java.util.*;
 import java.util.function.BiConsumer;
+import java.util.stream.Collectors;
 
+/**
+ * Implementation of {@link AttributeFetcher} that uses {@link SolrCloudManager}
+ * to access Solr cluster details.
+ */
 public class AttributeFetcherImpl implements AttributeFetcher {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-  boolean requestedNodeCoreCount;
-  boolean requestedNodeDiskType;
-  boolean requestedNodeFreeDisk;
-  boolean requestedNodeTotalDisk;
-  boolean requestedNodeHeapUsage;
-  boolean requestedNodeSystemLoadAverage;
-  Set<String> requestedNodeSystemPropertiesSnitchTags = new HashSet<>();
-  Set<String> requestedNodeMetricSnitchTags = new HashSet<>();
+  Set<String> requestedNodeSystemSnitchTags = new HashSet<>();
```

Comment:
I preferred the previous name `requestedNodeSystemPropertiesSnitchTags` ("system" by itself is too vague).

Reply:
Oh I see it's used for both system properties and environment variables... In which case I'd drop system in the name.

Reply:
I disagree, having "system" here makes it clearer what category of metrics we want to keep here. In any case, this is an internal variable name...